dev_docs/key_concepts/anatomy_of_a_plugin.mdx

```diff
@@ -39,27 +39,41 @@ plugins/
 
 ```
 {
-  "id": "demo",
-  "version": "kibana",
+  "id": "examplePluginId",
+  "version": "1.0.0",
+  "kibanaVersion": "7.14.0",
   "server": true,
   "ui": true,
-  "owner": { [1]
+  "owner": {
     "name": "App Services",
     "githubTeam": "kibana-app-services"
   },
-  "description": "This plugin extends Kibana by doing xyz, and allows other plugins to extend Kibana by offering abc functionality. It also exposes some helper utilities that do efg", [2]
-  "requiredPlugins": ["data"], [3]
-  "optionalPlugins": ["alerting"] [4]
-  "requiredBundles": ["anotherPlugin"] [5]
+  "description": "A description about this plugin!",
+  "requiredPlugins": ["data"],
+  "optionalPlugins": ["alerting"]
+  "requiredBundles": ["anotherPlugin"]
 }
 ```
 
-[1], [2]: Every internal plugin should fill in the owner and description properties.
+`id` - [Required] The id of your plugin can be anything, though it should be fairly unique, as every plugin in an installation needs to be unique. It must be snakeCase.
+
+`version` - [Required] Note the version of your plugin. Kibana doesn't use this field, but it is required.
+
+`kibanaVersion` - [Optional] If your plugin is only compatible with a specific version of Kibana, put it here. Note that internal, first-party plugins shouldn't use this, as they are expected to be compatible with every Kibana version.
+
+`server` - [Optional] If your plugin contains server-side code, this must be true.
+
+`ui` - [Optional] If your plugin contains client-side code, this must be true.
+
+`owner` - [Required] Help users of your plugin know who manages this plugin and how to get in touch. This is required for internal plugins.
```

Comment:
What's is a valid value here? Github team name that we use in CODEOWNERS? Do we have a check that a name is valid? Can there be several owners of a plugin? 
There are some comments, but they don't fully answer these questions
https://github.com/mshustov/kibana/blob/ca137ea44bcee3de7ce22d00c489b10168b020f3/src/core/server/plugins/types.ts#L232-L242

Reply:
updated, lmk what you think of this new version